### PR TITLE
Updated the changes when the user...

### PR DESCRIPTION
**Pull request by Surendhar1997**
... wants to register with same mail id more than once.

### CODE REVIEW

**Comment by Surendhar1997** [comment]
Also add changes during the registration process handled the case when no client name given in request for generation unique client code.

File: security/src/main/java/com/fincity/security/dao/ClientDAO.java [hunk 1]
```diff
@@ -524,4 +528,14 @@ public Mono<List<Package>> getPackagesAvailableForClient(ULong clientId) {
...
 	}
 
+	public Mono<List<ULong>> getManagingClientIds(List<ULong> list) {
+		
+		return Flux.from(this.dslContext.select(SECURITY_CLIENT_MANAGE.MANAGE_CLIENT_ID)
+				.from(SECURITY_CLIENT_MANAGE)
+				.where(SECURITY_CLIENT_MANAGE.CLIENT_ID.in(list)))
+				.map(Record1::value1)
+				.collectList()
+				.log();
```
Comment (serialcoder13) [inline security/src/main/java/com/fincity/security/dao/ClientDAO.java:538]: Remove Log

**Review by serialcoder13 — CHANGES_REQUESTED** [review]
Just remove the log in the method

**Review by serialcoder13 — APPROVED** [review]
LGTM

**Pull request merged**